src/platform/forms-system/src/js/web-component-fields/VaSelectField.jsx

```diff
@@ -20,7 +20,7 @@ export default function VaSelectField(props) {
     optionsList(props.childrenProps.schema);
   const labels = props.uiOptions?.labels || {};
 
-  if (!props.uiOptions?.uswds) {
+  if (!mappedProps?.uswds) {
```

Comment:
Got an error on my local /text-input-address page for VaSelectField, complaining about prop types: "...invalid; it must be a function... but received object"